### PR TITLE
Fix display of integrations parameters

### PR DESCRIPTION
**Pull request by yucohen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by julieschwartz18** [comment]
Doc review complete, minor updates. @yucohen - please review and confirm. Thanks!

**Review by julieschwartz18 — COMMENTED** [review]
Doc review complete, minor updates

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 1]
```diff
@@ -1,93 +1,61 @@
...
-The OData parameter can be used to create different queries for the ***msgraph-mail-list-emails*** and ***msgraph-mail-get-email*** commands. Please see [OData Docs](https://docs.microsoft.com/en-us/graph/query-parameters) for detailed information.
-Examples:
-* ```!msgraph-mail-list-emails user_id=dev@demistodev.onmicrosoft.com odata="$select=from"```
-* ```!msgraph-mail-list-emails user_id=dev@demistodev.onmicrosoft.com odata="$filter=from/emailAddress/address eq 'azure-noreply@microsoft.com'"```
-* ```!msgraph-mail-list-emails user_id=dev@demistodev.onmicrosoft.com odata="$filter=sentDateTime gt 2020-03-25T09:35:23Z and sentDateTime lt 2020-03-25T12:04:47Z" ```
-
-**NOTE:**
-The query parameter '$filter' is not supported when using the 'search' parameter.
+Microsoft Graph lets your app get authorized access to a user's Outlook mail data in a personal or organization account.
+This integration was integrated and tested with version xx of MicrosoftGraphMail
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:2]: Do you want to leave it as version xx?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 2]
```diff
@@ -1,93 +1,56 @@
...
-The OData parameter can be used to create different queries for the ***msgraph-mail-list-emails*** and ***msgraph-mail-get-email*** commands. Please see [OData Docs](https://docs.microsoft.com/en-us/graph/query-parameters) for detailed information.
-Examples:
-* ```!msgraph-mail-list-emails user_id=dev@demistodev.onmicrosoft.com odata="$select=from"```
-* ```!msgraph-mail-list-emails user_id=dev@demistodev.onmicrosoft.com odata="$filter=from/emailAddress/address eq 'azure-noreply@microsoft.com'"```
-* ```!msgraph-mail-list-emails user_id=dev@demistodev.onmicrosoft.com odata="$filter=sentDateTime gt 2020-03-25T09:35:23Z and sentDateTime lt 2020-03-25T12:04:47Z" ```
-
-**NOTE:**
-The query parameter '$filter' is not supported when using the 'search' parameter.
+Microsoft Graph lets your app get authorized access to a user's Outlook mail data in a personal or organization account.
+This integration was integrated and tested with version xx of MicrosoftGraphMail.
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:2]: Replace the xx  with a real value

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 3]
```diff
@@ -1,93 +1,56 @@
...
-
-## Email Attachments Limitations
-* The maximum attachment size to be sent in an email can be 150-MB. [large-attachments](https://docs.microsoft.com/en-us/graph/outlook-large-attachments?tabs=http)
-* The larger the attachment, the longer it would take for a command that supports adding attachments to run.
-* Requires the permission of Mail.ReadWrite (Application) - to send attachments > 3mb
-* When sending mails with large attachments, it could take up to 5 minutes for the mail to actually be sent.
-
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Server URL |  | True |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:12]: Don't you need a description for the url ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 4]
```diff
@@ -1,93 +1,56 @@
...
+    | Application ID or Client ID | See the Detailed Instructions section. | False |
+    | Token or Tenant ID | See the Detailed Instructions section. | False |
+    | Key or Client Secret | See the Detailed Instructions section. | False |
+    | Certificate Thumbprint | Used for certificate authentication. As appears in the "Certificates & secrets" page of the app. | False |
+    | Private Key | Used for certificate authentication. The private key of the registered certificate. | False |
+    | Fetch incidents |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:18]: Don't you need a description for this ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 5]
```diff
@@ -1,93 +1,56 @@
...
+    | Email address from which to fetch incidents | For example, "example@demisto.com" | False |
+    | Name of the folder or sub-folder from which to fetch incidents | Supports folder ID and sub-folders, for example Inbox/Phishing. | False |
+    | First fetch timestamp | &lt;number&gt; &lt;time unit&gt;, for example 12 hours, 7 days. | False |
+    | HTTP Timeout | The timeout of the HTTP requests sent to Microsoft Graph API (in seconds). | False |
+    | Maximum number of emails to pull per fetch |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:23]: Don't you need a description for this ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 6]
```diff
@@ -1,93 +1,56 @@
...
+    | Trust any certificate (not secure) |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:24]: Don't you need a description for this ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 7]
```diff
@@ -1,93 +1,56 @@
...
+    | Use system proxy settings |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:25]: Don't you need a description for this ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 8]
```diff
@@ -1,93 +1,56 @@
...
+    | Use a self deployed Azure application |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:26]: Don't you need a description for this ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 9]
```diff
@@ -1,93 +1,56 @@
...
+    | Incident type |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:27]: Don't you need a description for this ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 10]
```diff
@@ -1,93 +1,56 @@
...
+    | Display the entire email body | Whether to fetch incidents with the entire email body, or just an email body preview, mark if the full email body is required. |  |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:28]: Does not mention if it's required

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 11]
```diff
@@ -231,895 +129,434 @@ Returns the properties of an email.
...
+| MSGraphMail.Folders.DisplayName | String | The folder display name. | 
+| MSGraphMail.Folders.ID | String | The folder ID. | 
+| MSGraphMail.Folders.ParentFolderID | String | The parent folder ID. | 
+| MSGraphMail.Folders.TotalItemCount | Number | The total number of email messages in the folder. | 
+| MSGraphMail.Folders.UnreadItemCount | Number | The number of unread email messages in the folder. | 
+
+### msgraph-mail-create-folder
 ***
-Creates a new folder under the specified folder (parent).
+Creates a new folder under specified the specified folder (parent).
```
Comment (gal-forer) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:279]: Is this right ?

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 12]
```diff
@@ -1,558 +1,134 @@
...
-<tr>
-<td style="width: 292px;">MsGraph.Alert.Status</td>
-<td style="width: 29px;">string</td>
-<td style="width: 419px;">Alert status</td>
-</tr>
-</tbody>
-</table>
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Host URL |  | True |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:20]: Don't you need a description for this ?

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 13]
```diff
@@ -1,558 +1,134 @@
...
+    | Application ID or Client ID | See the Detailed Instructions section. | True |
+    | Token or Tenant ID | See the Detailed Instructions section. | True |
+    | Key or Client Secret | See the Detailed Instructions section. | False |
+    | Certificate Thumbprint | Used for certificate authentication. As appears in the "Certificates & secrets" page of the app. | False |
+    | Private Key | Used for certificate authentication. The private key of the registered certificate. | False |
+    | Trust any certificate (not secure) |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:26]: Don't you need a description for this ?

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 14]
```diff
@@ -1,558 +1,134 @@
...
+    | Use system proxy settings |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:27]: Don't you need a description for this ?

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 15]
```diff
@@ -1,558 +1,134 @@
...
+    | Use a self-deployed Azure application |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:28]: Don't you need a description for this ?

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 16]
```diff
@@ -1,558 +1,134 @@
...
+    | Fetch incidents |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:29]: Don't you need a description for this ?

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 17]
```diff
@@ -1,558 +1,134 @@
...
+    | Incident type |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:30]: Don't you need a description for this ?

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 18]
```diff
@@ -1,558 +1,134 @@
...
+    | First fetch time range | &lt;number&gt; &lt;time unit&gt;, for example 1 hour, 30 minutes | False |
+    | How many incidents to fetch each time |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:32]: Don't you need a description for this ?

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md [hunk 19]
```diff
@@ -26,27 +26,27 @@ Moreover, enter your client secret as the “Key” parameter and your client ID
...
-| first_fetch_delta | First fetch time range \(&lt;number&gt; &lt;time unit&gt;, e.g., 1 hour, 30 minutes\) | False |
-| content_types_to_fetch | Content types to fetch | False |
-| isFetch | Fetch incidents | False |
-| incidentType | Incident type | False |
-| record_types_filter | Record types to fetch \(Comma\-separated list of the record types you wish to fetch. Content records with a record  type that is not specified will not be fetched. If this field is left empty, all record types will be fetched.\) | False |
-| workloads_filter | Workloads to fetch \(Comma\-separated list of the workloads you wish to fetch. Content records with a workload that is not specified will not be fetched. If this field is left empty, all workloads will be fetched.\) | False |
-| operations_filter | Operations to \(Comma\-separated list of the operations you wish to fetch. Content records with an operation that is not specified will not be fetched. If this field is left empty, all operations will be fetched.\) | False |
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Base URL |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md:31]: Don't you need a description for this ?

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md [hunk 20]
```diff
@@ -26,27 +26,27 @@ Moreover, enter your client secret as the “Key” parameter and your client ID
...
+    | Application ID or Client ID | See the Detailed Instructions section. | False |
+    | Key or Client Secret | See the Detailed Instructions section. | False |
+    | Token or Tenant ID | See the Detailed Instructions section. | False |
+    | Certificate Thumbprint | Used for certificate authentication. As appears in the "Certificates & secrets" page of the app. | False |
+    | Private Key | Used for certificate authentication. The private key of the registered certificate. | False |
+    | Use a self-deployed Azure application |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md:37]: Don't you need a description for this ?

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md [hunk 21]
```diff
@@ -26,27 +26,27 @@ Moreover, enter your client secret as the “Key” parameter and your client ID
...
+    | Application redirect URI (for self-deployed mode) |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md:38]: Don't you need a description for this ?

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md [hunk 22]
```diff
@@ -26,27 +26,27 @@ Moreover, enter your client secret as the “Key” parameter and your client ID
...
+    | The authentication code you got for the service | For instructions on how to receive it, see the Detailed Instructions section. | False |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md:41]: Don't you need a description for this ?

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md [hunk 23]
```diff
@@ -26,27 +26,27 @@ Moreover, enter your client secret as the “Key” parameter and your client ID
...
+    | First fetch time range | &lt;number&gt; &lt;time unit&gt;, for example 1 hour, 30 minutes | False |
+    | Timeout | The default timeout (in seconds) for API calls. Default is 15 seconds. | False |
+    | Content types to fetch |  | False |
+    | Fetch incidents |  | False |
+    | Incident type |  | False |
```
Comment (gal-forer) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md:46]: Don't you need a description for this ?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 24]
```diff
@@ -1,93 +1,56 @@
...
+    | Fetch incidents |  | False |
+    | Email address from which to fetch incidents | For example, "example@demisto.com" | False |
+    | Name of the folder or sub-folder from which to fetch incidents | Supports folder ID and sub-folders, for example Inbox/Phishing. | False |
+    | First fetch timestamp | &lt;number&gt; &lt;time unit&gt;, for example 12 hours, 7 days. | False |
+    | HTTP Timeout | The timeout of the HTTP requests sent to Microsoft Graph API (in seconds). | False |
+    | Maximum number of emails to pull per fetch |  | False |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+    | Use a self deployed Azure application | Rather to use a selp-deployed application | False |
+    | Incident type |  | False |
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:27]: What should the description for Incident type be? If this is a string for the incident type, whey is the default value false?

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md [hunk 25]
```diff
@@ -1,93 +1,56 @@
...
+    | Display the entire email body | Whether to fetch incidents with the entire email body, or just an email body preview, mark if the full email body is required. | False |
+    | Advanced: Time in minutes to look back when fetching emails | Use this parameter to determine how long backward to look in the search for incidents that were created before the last run time and did not match the query when they were created. | False |
```
Comment (julieschwartz18) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md:29]: If this is time in minutes to look back, why is the default value False?

**Review by gal-forer — APPROVED** [review]

**Pull request merged**